Formula/c/cairo.rb

```diff
@@ -27,11 +26,11 @@ class Cairo < Formula
 
   head do
     url "https://gitlab.freedesktop.org/cairo/cairo.git"
```

Comment:
This doesn't need to be a block now. Please use:

```ruby
head "https://gitlab.freedesktop.org/cairo/cairo.git", branch: "master"
```

And move it to below the `license` line.